MyApplication/app/src/main/java/com/example/ds/userinfotabs/MyInfoPagerActivity.java

```diff
@@ -0,0 +1,49 @@
+package com.example.ds.userinfotabs;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+
+/**
+ * Created by ds on 11/1/16.
+ */
+
+public class MyInfoPagerActivity extends FragmentPagerAdapter {
```

Comment:
I suggest not putting "Activity" in a class name unless that class is actually an activity itself. Having "activity" in this name could lead to confusion if you were working on a team.